app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
I started to write a test in the partial view testing that the first name showed up, but I started thinking it wasn't necessary (pseudocode-ish):

```
view_model = ProfileIndex.new(decrypted_pii: first_name: 'name')
greeting = view_model.decrypted_pii.first_name
allow(view) to receive(greeting).and_return(greeting)

expect(rendered).to have_content(decypted_pii.first_name)
```
Something like this doesn't exactly test that the view is going to behave properly when the user is verified, just that the first name appears when provided to the view (via the view model). Maybe that is the same thing?